src/index.ts

```diff
@@ -12,6 +12,7 @@ export { Modal, ModalWrapper, ModalTrigger, ModalEraser, ModalContent } from './
 export { Pagination } from './components/Pagination'
 export { Radio } from './components/Radio'
 export { RadioLabel } from './components/RadioLabel'
+export { Button } from './components/Button'
```

Comment:
It may not be related to your use, but can you export ButtonAnchor and ButtonDiv together?